queries/highlights.scm

```diff
@@ -6,7 +6,7 @@
 ; Includes
 
 ((symbol) @include
-  (#match? @include "include"))
+  (#match? @include "^include$"))
```

Comment:
I just see that `#match?` is/was used here, I think it would be better to use `#eq?` instead.
Also this should be limited to the first list element, so I guess it should look like this:

```suggestion
(list .
  ((symbol) @include
    (#eq? @include "include")))
```

Reply:
Right, I forgot that `#eq?` exists, that's much better now